src/test/java/SimpleCalculatorTest.java

```diff
@@ -0,0 +1,62 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import simpleCalc.SimpleCalculator;
+
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("초간단 계산기 unit 테스트")
+public class SimpleCalculatorTest {
+
+    @Nested
+    @DisplayName("사칙 연산 계산 기능 테스트")
+    class FourOperationTest {
+
+        @Test
+        @DisplayName("add 함수는 두 정수를 더한 값을 반환한다.")
+        void testAddOperation() {
+            assertThat(SimpleCalculator.add(1, 2)).isEqualTo(3);
+            assertThat(SimpleCalculator.add(-1, -2)).isEqualTo(-3);
+            assertThat(SimpleCalculator.add(2, 1)).isEqualTo(3);
+            assertThat(SimpleCalculator.add(1, -1)).isEqualTo(0);
+        }
```

Comment:
팁) 이렇게 여러 테스트 케이스를 한 메서드로 만들고 싶다면 `@ParameterizedTest`를 사용하는게 더 나을거에요

```suggestion
        @ParameterizedTest(name = "{0} add {1} 은 {2}이다")
        @CsvSource({"1,2,3", "-1,-2,-3", "2,1,3", "1,-1,0"})
        @DisplayName("add 함수는 두 정수를 더한 값을 반환한다.")
        void testAddOperation(int a, int b, int expected) {
            assertThat(SimpleCalculator.add(a, b)).isEqualTo(expected);
        }
```

만약 케이스가 여러개인 것이 아니라 한 케이스에 대해 여러 검증문을 검증하고 싶다면 다음과 같은 방법을 사용할 수 있어요!

```suggestion
        assertAll(
                    () -> assertThat(SimpleCalculator.add(1, 2)).isEqualTo(3),
                    () -> assertThat(SimpleCalculator.add(-1, -2)).isEqualTo(-3),
                    () -> assertThat(SimpleCalculator.add(2, 1)).isEqualTo(3),
                    () -> assertThat(SimpleCalculator.add(1, -1)).isEqualTo(0),
            );
```

assertAll같은 경우는, 원래 검증문이 여러개일 때 검증문이 실패할 경우 그 뒤의 검증문은 실행되지 않고 그대로 종료됩니다! 그걸 방지하고 전체 검증문을 모두 돌린 뒤 실패 케이스를 모두 표시해주는 역할을 해요. (전문 용어로 soft assertion 이라고 합니다)